plugins/restful/RestfulFilesUpload.php

```diff
@@ -26,6 +26,8 @@ public static function controllersInfo() {
    * - "validators": By default no validation is done on the file extensions or
    *   file size.
    * - "scheme": By default the default scheme (e.g. public, private) is used.
+   * - "make_permanent": When true the file's status will be set to permanent
```

Comment:
maybe this can be called ``status`` and the default value be ``FILE_STATUS_PERMANENT``?